benchexec/tools/validation_utils.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module contains some useful functions for handling Validation Tasks
+"""
+
+from pathlib import Path
+
+from benchexec.tools.template import UnsupportedFeatureException
+import benchexec.tools.template
+
+
+def __partition_input_files(task):
```

Comment:
The Python way to name private stuff is with a single underscore as prefix.

Reply:
Thanks a lot for the comment. Has been changed.